### PR TITLE
Testing enumerable methods 

### PR DESCRIPTION
**Pull request by helman101**
## Rspec
- Using rspec to test all our enumerable methods
- Used describe for taking the module and each of the methods
- Used it for declaring each of the tests
- Used expect to check the method
- Used .to eql() to declare the expected result

### CODE REVIEW

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Captain Kumar & Andres Ortegon Pulido** 
Good job with your first submission for this project :muscle:  :muscle:
It meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [x] Don't edit the rubocop file.
- [x] Test both positive and negative scenarios for each tested method. You might use "to_not eql" or "to_not be_an".
- [x] Add a screenshot in your README file.

#### OPTIONAL

- [x] In the enum_method_spec.rb, use "require_relative" instead or "require".

Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all project requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
